tests/README.md

```diff
@@ -18,9 +18,10 @@ This directory contains the functional testing suite for stacker. It exercises a
   ```console
   $ ./stacker.yaml.sh | stacker build -
   ```
-4. In the AWS console, generate a new IAM access key pair for the user and set it in your shell:
+4. Grab the generated key pair for the user and set it in your shell:
 
   ```console
+  $ ./stacker.yaml.sh | stacker info -
```

Comment:
Hah, this is literally a question I had in my "todo" list at work:

https://www.dropbox.com/s/5tgbm4jl7hd343b/Screenshot%202017-09-08%2013.49.55.png?dl=0

@ejholmes what do you think?

Reply:
I have an adverse reaction to putting secrets in CloudFormation outputs (plaintext, bad ACL's), but given the user already has a pretty scoped set of permissions, it's prob ok.

Reply:
The goal was to avoid having to go to the console or do an API call to grab the key. As @ejholmes  said, since the user is meant to have limited perms. and be temporary it shouldn't be terribly risky.